float32 scale_max
float32 scale_min
---
